github-pullrequest-plugin/src/main/resources/org/jenkinsci/plugins/github/pullrequest/GitHubPRTrigger/help-skipFirstRun.html

```diff
@@ -1,4 +1,7 @@
 <div>
-    If this checked, then first trigger run will not run real builds.
-    After first run this value automatically unset.
+    If selected, PRs that were pending before a first dry run of this job will
+    be ignored, and only PRs updated or created after that first dry run will
+    be built.
+    <br/>
+    NOTE: After selecting this option, you need to run the build once by hand.
```

Comment:
https://github.com/jenkinsci/github-integration-plugin/blob/master/github-pullrequest-plugin/src/main/java/org/jenkinsci/plugins/github/pullrequest/GitHubPRTrigger.java#L356-L373
Not really "ignore PRs" it would ignore triggering builds. Yes it should be automatically unset after first run.